ow_faults_detection/include/ow_faults_detection/FaultDetector.h

```diff
@@ -85,32 +81,23 @@ class FaultDetector
   ros::Publisher m_power_faults_msg_pub;
   ros::Publisher m_system_faults_msg_pub;
 
-  // Arm and antenna
-  ros::Subscriber m_joint_states_sub;
-
-  // Camera
+  // faults topic subscribers;
+  ros::Subscriber m_joint_states_sub; // for arm and antenna
   ros::Subscriber m_camera_original_trigger_sub;
   ros::Subscriber m_camera_raw_sub;
-
-  // Power
   ros::Subscriber m_power_soc_sub;
   ros::Subscriber m_power_temperature_sub;
```

Comment:
Unrelated to the intended change, but these members are not necessary since they are are used only to hold a value and are never referenced individually after assignment.   They could be replaced with a single member: std::list<ros::Subscriber>, and each subscriber can simply be pushed into this list.